### ISSUE TITLE
why are you hunting job?

### ISSUE THREAD

**Issue by monkindey** [egoist/ama#11]
you are so productive and it will be ok to join in the company like BAT.

**Comment by egoist** [comment]
B and A do no seem to have front-end team in Chengdu, and I didn't pass T's interview.

And the situation for me is I have to stay in Chengdu for at least 2 years for some reason, so I'm looking for job in other companies here.

**Comment by monkindey** [comment]
I can't believe you failed in the T's interview. What's the requirement of it. Maybe you can find some remote working on the stackoverflow jobs columns. I have no doubt that you can make it.

**Comment by egoist** [comment]
Nah, not from Stack Overflow, I suck at English 😂

**Comment by egoist** [comment]
Honestly, I'm just a *hobby* programmer with little knowledge on data structure & algorithms.

**Comment by monkindey** [comment]
yeah, the T's interview will involve the basic knowledge, such as network, os, structure etc. Good luck to you! you can make it done.
